Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.7-slim
+
+LABEL description.short="DACCS image updater"
+LABEL description.long="DACCS image updater"
+LABEL maintainer="Mathieu Provencher <mathieu.provencher@crim.ca>"
+LABEL vendor="CRIM"
+LABEL version="0.1"
+
+# setup paths
+ENV APP_DIR=/opt/local/src/daccs-image-updater
+ENV TEST_DIR=${APP_DIR}/tests/integration
+WORKDIR ${APP_DIR}
+
+# install package dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget \
+    jq \
+    git \
+    curl \
+    lsof
+
+# install GitHub's hub CLI tool
+RUN wget -O hub.tgz --progress=dot:mega https://github.com/github/hub/releases/download/v2.14.2/hub-linux-amd64-2.14.2.tgz
+RUN mkdir /hub
+RUN tar -xvf hub.tgz -C /hub --strip-components 1
+RUN bash /hub/install
+
+# configure github account. Defaults, overridden by GITHUB_USER
```

Comment:
GITHUB_USER will override both the user.email and user.name of git config?  Unclear in comment.

Reply:
GITHUB_USER and GITHUB_PASSWORD have been replaced by GITHUB_TOKEN